patches/react-native-gifted-charts+1.2.42.patch

```diff
@@ -10,7 +10,48 @@ index 8ef5ecc..4d11a61 100644
    let data = useMemo(() => {
      if (!props.data) {
        return [];
-@@ -2952,6 +2953,8 @@ export const LineChart = (props: propTypes) => {
+@@ -820,30 +821,36 @@ export const LineChart = (props: propTypes) => {
+     // console.log('comes here............')
```

Comment:
Just a question about these patch files, did you make these yourself, or are these suggested from official docs? Cos these feel like a lot of changes for what a patch file should be

Also there's a console.log to remove